requirements-docs.txt

```diff
@@ -12,7 +12,7 @@ sphinx-reredirects
 sphinx-sitemap
 sphinxcontrib.httpdomain
 sphinxcontrib.httpexample
-sphinxcontrib-mermaid==0.8.1 # See https://github.com/mgaitan/sphinxcontrib-mermaid/issues/110#issuecomment-1473688354
+sphinxcontrib-mermaid
```

Comment:
sphinxcontrib.mermaid

Reply:
Are you sure? See https://pypi.org/project/sphinxcontrib-mermaid/, plus using a dash works.

> ## Installation
> You can install it using pip
>
> `pip install sphinxcontrib-mermaid`

Reply:
Also in the install instructions in the docs:

> extensions = [
>     ...,
>     'sphinxcontrib.mermaid'
> ]